internal/apis/certmanager/types_certificate.go

```diff
@@ -178,6 +178,11 @@ type CertificateSpec struct {
 	// `--feature-gates=AdditionalCertificateOutputFormats=true` option on both
 	// the controller and webhook components.
 	AdditionalOutputFormats []CertificateAdditionalOutputFormat
+
+	// CleanupPolicy is when this field is set to `Delete`, the owner reference is always created on the Secret resource and the secret will be automatically removed when the certificate resource is deleted.
+	// When this field is set to `Never`, the owner reference is never created on the Secret resource and the secret will not be automatically removed when the certificate resource is deleted.
+	// If the value of this field is unset this field "inherits" the value of the flag `--enable-certificate-owner-ref`.
```

Comment:
Wrapping to 80 chars like in the rest of the file would greatly help reviewing and reading that comment 👍

Reply:
Thank you, i wrapping comments to 80 chars.